x-pack/test/apm_api_integration/tests/services/throughput.ts

```diff
@@ -100,7 +100,7 @@ export default function ApiTest({ getService }: FtrProviderContext) {
 
         it('has the correct throughput in tpm', () => {
           const avg = mean(throughputResponse.currentPeriod.map((d) => d.y));
-          expectSnapshot(avg).toMatchInline(`6.19354838709677`);
+          expectSnapshot(avg).toMatchInline(`0.240860215053763`);
```

Comment:
The changes here are very big. Can you double check that it's correct?

Reply:
<img width="849" alt="Screen Shot 2021-09-20 at 13 10 34" src="https://user-images.githubusercontent.com/55978943/134182452-0552fb09-8d4d-4c7f-a60a-38a25d059589.png">